include/irr/asset/IShader.h

```diff
@@ -20,6 +20,13 @@ namespace irr
 namespace asset
 {
 
+//! Interface class for Unspecialized Shaders
+/*
+	The purpose for the class is for storing raw
+	glsl code. Such a shader has to be passed
```

Comment:
it can actually store both raw GLSL to be compiled or already compiled (but unspecialized) SPV

Reply:
adjusted in following commit